tests/acceptance/features/coreApiCapabilities/capabilities.feature

```diff
@@ -9,7 +9,7 @@ Feature: capabilities
     When the administrator retrieves the capabilities using the capabilities API
     Then the OCS status code should be "100"
     And the HTTP status code should be "200"
-    And the JSON data of the response should match
+    And the ocs JSON data of the response should match
```

Comment:
required in this PR?
```suggestion
    And the JSON data of the response should match
```

Reply:
Yes, otherwise the newly added step matches the current step and there will be conflicts between two steps.

Reply:
But we don't have `the ocs JSON data of the response should match` step implementation yet.